sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Features Added
 
+- Added ability to track progress by passing `ProgressReporter` in the `Context`. 
+  I.e., `Contexts.with(context).setProgressReporter(progressReporter)`
+  before calling `HttpClient.send(HttpRequest, Context)` API.
```

Comment:
Two things:

1. s/ProgressReporter/ProgressListener
2. Just to be clear - the user would need to `getContext()` and pass that in, rather than expect to mutate the existing `Context` passed in to the `with` method with the additional progressReporter. I sat here for a second thinking "do we have two different usage patterns here?" before concluding that the naive (and incorrect, but only with knowledge about how Context works) interpretation of the code sample above would be to do the following:

```java
Context ctx = Context.NONE;
Contexts.with(ctx).setProgressListener(progressListener);
httpClient.send(request, ctx);
```

Reply:
Changed to sample, see effect here https://github.com/kasobol-msft/azure-sdk-for-java/blob/progress-reporter/sdk/core/azure-core-http-netty/CHANGELOG.md#features-added